aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAemMojo.java

```diff
@@ -34,6 +34,7 @@
 import org.eclipse.aether.resolution.ArtifactResolutionException;
 import org.eclipse.aether.resolution.ArtifactResult;
 
+import com.adobe.aem.analyser.mojos.extensions.DependencyLifecycleParticipant;
```

Comment:
Why is this import needed?

Reply:
Because I moved all extensions to a dedicated subpackage `com.adobe.aem.analyser.mojos.extensions`. It used to be in the same package as this Mojo before and therefore no import was necessary.